changelog.md

```diff
@@ -1,3 +1,95 @@
+### v3.0.0
+
+Version 3.0.0 of Marionette has arrived and contains many improvements over version
+2.x but also some API Changes. Below is a list of the changes made to each component.
+
+To help the community transition over we have released a v2 patch tool to assist
+the upgrade. [Marionette Patch Tool] (https://github.com/marionettejs/marionette-v3-compat)
+
+#### View
+* `LayoutView` + `ItemView` merge and rename to `View`.
+* `Marionette.View` -> `ViewMixin`
+* Add `LayoutView` shortcut methods
+* `isDestroyed` and `isRendered` need to be made private with a public accessor method.
+* Add `before:show` event to `ChildView` when added after show
+* Call `Backbone.View` with result of options (163188eeb8)
+* Make `CompositeView`'s `renderChildren` public.
+* Rename `childEvents` to `childViewEvents`.
+* View: set `isDestroyed` to false by default breaking
+* Remove passing view options as a function breaking
+* Rename `templateHelpers` to `templateContext`
+* Make sure `before:render` is triggered before emptying regions.
+* Regions are not attached directly to the layout. Use `getRegion` instead.
+* Allow `CompositeView` to attach to existing HTML with `template:false`
+* Add `hasRegion` for layouts
+* Make `getChildView` private
```

Comment:
Oh also.. this is for CollectionView..  On a regular view `getChildView` is the same as in `v2.x` nothing was changed or privatized.